### PR TITLE
fix(#38): switch from `docker-compose` to `docker compose`

### PR DESCRIPTION
**Pull request by jkuester**
Closes https://github.com/medic/cht-upgrade-service/issues/38

### CODE REVIEW

File: test/e2e/install.spec.js [hunk 1]
```diff
@@ -106,11 +106,8 @@ describe('install', () => {
       'one-two.yml': await utils.setVersion('one-two.yml', '13.0.0', false),
     })).to.be.rejected;
     expect(response.error).to.equal(true);
-    expect(response.reason).to.include(
```
Comment (jkuester) [inline test/e2e/install.spec.js:109]: Newer versions of `docker compose` (and `docker-compose` for that matter...) seem to now only report one of the images that fails to pull (verified this behavior locally).  My guess is that there is some indeterminate race-conditions as to which will missing image will be reported....

**Comment by jkuester** [comment]
@Hareet not sure if you had something else in flight for this issue, but I figured I would at least get this PR to build if we wanted to just swap to `docker compose`.  Otherwise, we can just close this and move on. :+1:

File: src/docker-compose-cli.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 const childProcess = require('child_process');
 
-const DOCKER_COMPOSE_CLI = 'docker-compose';
+const DOCKER_CLI = 'docker';
```
Comment (dianabarsan) [inline src/docker-compose-cli.js:3]: Is there any sense to declare this here and not declare the `compose` on line 13?

**Review by dianabarsan — APPROVED** [review]
Awesome work and great to see you on this so quickly. I only added a very minor stylistic comment below.

**Comment by jkuester** [comment]
@dianabarsan regarding next steps for this, what needs to happen for this to get released?  How do updates for the cht-upgrade-service actually get pushed to live deployments?  

I am primarily concerned that we have some plan/documentation for any manual steps that might need to happen for someone already running Docker 26. With a little pointing in the right direction in regards to the deployment process, I would be happy to help with testing/documentation. (And @mrjones-plip has indicated the same!)

**Comment by dianabarsan** [comment]
> How do updates for the cht-upgrade-service actually get pushed to live deployments?

They do not. With the upgrade-service we don't rely on versions, but we do use the `latest` tag in our docker-compose, so all people need to do is do a `docker-compose pull` and `docker-compose up`.

**Review by dianabarsan — APPROVED** [review]
Is there a way to run a regex command? :D :D :D 

```
sh> docker[\s|-]compose up
```

**Comment by jkuester** [comment]
>  so all people need to do is do a docker-compose pull and docker-compose up.

Got it.  Do we have a way to notify folks that an update is available?  Do we just need a forum post or something?  Seems like we need to communicate:

- Admins need to take manual steps to update their cht-upgrade-service _before upgrading to Docker 26_.
- Also, we should capture the recovery steps for what to do if you broke everything by trying to do an upgrade on Docker 26 with the old cht-upgrade-service
- Anything else?

**Comment by jkuester** [comment]
Also, am I correct in assuming this does not affect k8s (since no `docker-compose` is involved)?

**Comment by dianabarsan** [comment]
I suppose that updating our documentation (which is suppose is this: https://docs.communityhealthtoolkit.org/apps/guides/hosting/4.x/self-hosting/single-node/ ) to include a section about upgrading the upgrade service, and a forum post about Docker 26 compatibility would be ok. 

> Also, am I correct in assuming this does not affect k8s

Correct.

**Pull request merged**